corehq/messaging/scheduling/scheduling_partitioned/models.py

```diff
@@ -331,8 +334,15 @@ def send_current_event_content_to_recipients(self):
 
         logged_event = MessagingEvent.create_from_schedule_instance(self, content)
 
+        def log_filter_error(filtered_recipient, msg):
+            sub_event = logged_event.create_subevent_from_contact_and_content(
+                filtered_recipient,
+                content,
+            )
+            sub_event.error(MessagingEvent.ERROR_FILTER_MISMATCH, msg)
```

Comment:
In the use case here, should this always be categorized as an "error"? Aren't there situations where the user data filter doesn't match but it's also expected to not match?

Reply:
renamed it to just FILTER_MISMATCH

Reply:
nice, i also meant `sub_event.error` when there's no status argument results in `MessagingEvent.STATUS_ERROR`. Should we use status `STATUS_NOT_COMPLETED` instead?

Reply:
I am not quite sure about this one. I agree error might not be the best phrasing but it looks like not completed is somewhat used like in progress in the sms code. So I am somewhat hesitant to use it instead. Error is clearly a final state. So that seems saver.